### PR TITLE
feat: git hooks (pre-commit)

### PR DESCRIPTION
**Pull request by cv65kr**
# Reason for This PR

During the development I am always forgetting to check code by linters.

## Description of Changes

Pre commit script

## License Acceptance

By submitting this pull request, I confirm that my contribution is made under the terms of the MIT license.

## PR Checklist

`[Author TODO: Meet these criteria.]`
`[Reviewer TODO: Verify that these criteria are met. Request changes if not]`

- [x] All commits in this PR are signed (`git commit -s`).
- [x] The reason for this PR is clearly provided (issue no. or explanation).
- [x] The description of changes is clear and encompassing.
- [x] Any required documentation changes (code and docs) are included in this PR.
- [x] Any user-facing changes are mentioned in `CHANGELOG.md`.
- [x] All added/changed functionality is tested.

### CODE REVIEW

File: .githooks/pre-commit [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+set -e -o pipefail
+
+if ! command -v golangci-lint &> /dev/null; then
+    echo "golangci-lint is not installed"
+    exit 1
+fi
+
+exec golangci-lint run "$@"
```
Comment (rustatian) [inline .githooks/pre-commit]: `golangci-lint --build-tags=race run`

Comment (cv65kr) [reply .githooks/pre-commit]: Do you want to check all files all the time? Even if commiter even not touch it?

Comment (rustatian) [reply .githooks/pre-commit]: Golangci-lint caches files. But `--build-tags=race` is about the internal RR's flag for the unsafe buffer check and fast string<->[]byte conversion: https://github.com/roadrunner-server/sdk/blob/master/utils/race_checker.go

Comment (rustatian) [reply .githooks/pre-commit]: I mean, add this flag to the command, like: `exec golangci-lint --build-tags=race run "$@"`

Comment (cv65kr) [reply .githooks/pre-commit]: @rustatian  got it, updated

**Comment by rustatian** [comment]
Thanks @cv65kr 👍🏻

**Pull request merged**